api/proto/teleport/legacy/types/types.proto

```diff
@@ -5480,6 +5480,9 @@ message PluginOktaSettings {
 
   // OrgUrl is the Okta organization URL to use for API communication.
   string org_url = 1;
+
+  // Version is the version of the Okta settings encoded in this resource.
+  string version = 2;
```

Comment:
I thought we would just use V2 when creating an Okta Plugin resource but I can see how it could be confusing if we set V2 on PluginV1 resource.

TBH I think having a separate version for settings is also confusing because it becomes hard to understand what the difference is between different versions. How about we just add another explicit property like "bool user_sync_enabled" or something like that instead?